python/chronos/src/bigdl/chronos/forecaster/base_forecaster.py

```diff
@@ -289,14 +308,50 @@ def fit(self, data, epochs=1, batch_size=32):
                                                 torch.from_numpy(data[1])),
                                   batch_size=max(1, batch_size//self.num_processes),
                                   shuffle=True)
-
             # Trainer init and fitting
-            self.trainer = Trainer(logger=False, max_epochs=epochs,
-                                   checkpoint_callback=self.checkpoint_callback,
-                                   num_processes=self.num_processes, use_ipex=self.use_ipex,
-                                   distributed_backend="spawn")
-            self.trainer.fit(self.internal, data)
-            self.fitted = True
+            if not val_data:
+                self.trainer = Trainer(logger=None, max_epochs=epochs,
+                                       checkpoint_callback=self.checkpoint_callback,
+                                       num_processes=self.num_processes, use_ipex=self.use_ipex,
+                                       flush_logs_every_n_steps=10, log_every_n_steps=10,
+                                       distributed_backend="spawn")
+                self.trainer.fit(self.internal, data)
+                self.fitted = True
+            elif val_data and val_mode == 'output':
```

Comment:
we don't need to make 'output' mode special since we will return this val_dict for all 3 modes.